src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,45 @@
-export const Tabs = () => {};
+import React from "react";
+import { Tab, TabProps } from "./Tab";
```

Comment:
Add this file

Reply:
if you have questions, feel free to ask in the chat

Reply:
Please push all files. I can't see where you get TabProps